5.5/mysql/bin/entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+
+function usage {
+	echo "You must specify following environment variables:"
+	echo "  \$MYSQL_ROOT_PASSWORD"
+	echo "  \$MYSQL_USER"
+	echo "  \$MYSQL_PASSWORD"
+	echo "  \$MYSQL_DATABASE"
+	exit 1
+}
+
+test -z "$MYSQL_ROOT_PASSWORD" && usage
+test -z "$MYSQL_USER" && usage
+test -z "$MYSQL_PASSWORD" && usage
+test -z "$MYSQL_DATABASE" && usage
+
+export X_SCLS="mysql55"
+source /opt/rh/mysql55/enable
+
+if [ ! -d '/var/lib/mysql/data' -a "$1" = 'mysqld_safe' ]; then
+
+	echo 'Running mysql_install_db ...'
+	mysql_install_db --user=mysql --datadir=/var/lib/mysql/data
+	echo 'Finished mysql_install_db'
+
+	# These statements _must_ be on individual lines, and _must_ end with
+	# semicolons (no line breaks or comments are permitted).
+	# TODO proper SQL escaping on ALL the things D:
+	TEMP_FILE='/tmp/mysql-first-time.sql'
+	cat > "$TEMP_FILE" <<-EOSQL
+		DELETE FROM mysql.user ;
+		CREATE USER 'root'@'%' IDENTIFIED BY '${MYSQL_ROOT_PASSWORD}' ;
```

Comment:
I see... are we going to give our users two mysql users? root && normal user? I would go just with one.